### DISCUSSION TITLE
Use unit abbreviation in Gridview

### DISCUSSION THREAD

**Discussion by polymed** [pimcore/pimcore#8064] [Q&A / Support] [answered]
Is there already a way to configure a QuantityValueInput field to use the unit abbreviation? I would just use the abbreviation as ID, but a unit must have a numeric ID!

Best regards 

![image](https://user-images.githubusercontent.com/51081904/106748429-82403080-6625-11eb-8379-df227039c57a.png)

**Comment by dvesh3** [comment, marked as answer]
I think this behavior is wrong and should be fixed for Grid view. please see https://github.com/pimcore/pimcore/issues/8095

**Reply by dvesh3** [reply]
@polymed fixed for v6.8.8 https://github.com/pimcore/pimcore/pull/8098

**Reply by polymed** [reply]
Thanks a lot
